Assets/UI/Panels/citypanel.lua

```diff
@@ -1496,13 +1660,15 @@ function SetupCollapsibleToggle( pCheckBoxControl:table, pButtonControl:table, p
     pCheckBoxControl:RegisterCheckHandler(
         function()
             pAreaControl:SetHide( pCheckBoxControl:IsChecked() );
-        end);
+        end
```

Comment:
this and next minor change are just to match the code in the existing unmodified Firaxis version of this file